📝 fix typo in date.md

bennettp123 opened the pull request:
<!-- Context of the PR: short description and potentially linked issues -->

<!-- ...a few words to describe the content of this PR...               -->
<!-- ... -->

fix typo in the doco for date arbitraries at https://fast-check.dev/docs/core-blocks/arbitraries/primitives/date/

<!-- Type of PR: [ ] unchecked / [ ] checked -->

**_Category:_**

- [ ] ✨ Introduce new features
- [x] 📝 Add or update documentation
- [ ] ✅ Add or update tests
- [ ] 🐛 Fix a bug
- [ ] 🏷️ Add or update types
- [ ] ⚡️ Improve performance
- [ ] _Other(s):_ ...
  <!-- Don't forget to add the gitmoji icon in the name of the PR -->
  <!-- See: https://gitmoji.dev/                                  -->

<!-- Fixing bugs, adding features... may impact existing ones           -->
<!-- in order to track potential issues that could be related to your PR -->
<!-- please check the impacts and describe more precisely what to expect -->

**_Potential impacts:_**

<!-- Generated values: Can your change impact any of the existing generators in terms of generated values, if so which ones? when? -->
<!-- Shrink values:    Can your change impact any of the existing generators in terms of shrink values, if so which ones? when? -->
<!-- Performance:      Can it require some typings changes on user side? Please give more details -->
<!-- Typings:          Is there a potential performance impact? In which cases? -->

- [ ] Generated values
- [ ] Shrink values
- [ ] Performance
- [ ] Typings
- [ ] _Other(s):_ ...

dubzzz commented on website/docs/core-blocks/arbitraries/primitives/date.md:
```diff
@@ -4,7 +4,7 @@ slug: /core-blocks/arbitraries/primitives/date/
 # Date
 
-Generate boolean values.
+Generate Date values.
```
```suggestion
Generate date values.
```

dubzzz reviewed (commented):
Thanks a lot for the contribution. As other "Generate xxx values" use lowercase, I'd do the same on this one.

Additionally, you'll have to run `yarn && yarn version:bump` on this PR (mark fast-check has patched and all others as declined)

bennettp123 commented:
> Thanks a lot for the contribution. As other "Generate xxx values" use lowercase, I'd do the same on this one.
> 
> Additionally, you'll have to run `yarn && yarn version:bump` on this PR (mark fast-check has patched and all others as declined)

Thanks—I've updated the PR to include your suggestion to use lowercase :+1:

I've also added a version bump, but I'm not sure I've done it correctly—`yarn && yarn version:bump` doesn't seem to do anything at all, so I committed the results of the command below:

```sh
yarn \
  && yarn workspaces foreach --all --exclude fast-check version decline --deferred \
  && yarn workspace fast-check version patch --deferred
```

dubzzz reviewed (approved):
Perfect

dubzzz commented:
Thanks a lot for this contribution.
